e2e/instance_test.go

```diff
@@ -38,6 +41,7 @@ func testInstance(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, testInstanceHash, resp.Hash)
 		require.Equal(t, testServiceHash, resp.ServiceHash)
+		require.Equal(t, hash.Dump([]string{"BAR=2", "FOO=1", "REQUIRED", "BAR=3", "REQUIRED=4"}), resp.EnvHash)
```

Comment:
this test should be replaced by:
```suggestion
		require.Equal(t, hash.Dump([]string{"BAR=3", "FOO=1", "REQUIRED=4"}), resp.EnvHash)
```
we don't want multiple env with the same key.